images/win/scripts/Tests/WindowsFeatures.Tests.ps1

```diff
@@ -11,15 +11,25 @@ Describe "WindowsNETFeatures" {
         $testCases += @{ FeatureName = "DSC-Service" }
     }
 
-    It "Windows NET Feature <FeatureName> is installed" -TestCases $testCases {
+    It "Windows Feature <FeatureName> is installed" -TestCases $testCases {
         (Get-WindowsFeature -Name $FeatureName).InstallState | Should -Be "Installed"
     }
-
-
 }
 
 Describe "ContainersFeature" {
     It "Windows containers feature is installed" {
         (Get-WindowsFeature -Name "Containers").InstallState | Should -Be "Installed"
     }
+}
+
+if (Test-isWin19) {
+    Describe "WSL" {
```

Comment:
I think it is simple to just add WSL to the list above and avoid code duplication.
```
$testCases = @(
        @{ FeatureName = "NET-Framework-Features" }
        @{ FeatureName = "NET-Framework-45-Features" }
        @{ FeatureName = "FS-iSCSITarget-Server" }
    )

    if (Test-isWin16) {
        $testCases += @{ FeatureName = "BITS" }
        $testCases += @{ FeatureName = "DSC-Service" }
    }
    if (Test-isWin19) {
        $testCases += @{ FeatureName ="Microsoft-Windows-Subsystem-Linux" }
    }

Reply:
done